baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -105,6 +109,11 @@ export class AssetServiceClient {
     // Save the auth object to the client, for use by other methods.
     this.auth = (this._gaxGrpc.auth as gax.GoogleAuth);
 
+    // Set the default scopes in auth client if needed.
+    if (servicePath === staticMembers.servicePath) {
```

Comment:
that's a nice simple approach 👍 it seems like enough of an edge case that someone would purposefully override `sevicePath`, with the same value, that I'd say let's go with this approach and cross that bridge if it ever happens.